chain-vote/src/committee.rs

```diff
@@ -95,10 +95,7 @@ impl Bech32 for ElectionPublicKey {
 #[derive(Clone)]
 pub struct MemberState {
     sk: MemberSecretKey,
-    owner_index: usize,
     apubs: Vec<GroupElement>,
-    es: Vec<GroupElement>,
-    encrypted: Vec<(HybridCiphertext, HybridCiphertext)>,
```

Comment:
It's fine if we just want to get rid of the warnings, but I think it would make more sense to either remove the code that generates shares altogether or ignore the warning as to signal this is a temporary implementation and the fields would need to be used in the future

Reply:
What are these fields going to be used for? Decryption key recovery?

Reply:
This is the first part of the DKG protocol. Those are the values you're supposed to share at the end of the first phase, but without the rest of the protocol they are not really too helpful.
However, if we are going to remove them we might as well remove all the code that generates them

Reply:
Right, I will override the lint instead.